controllers/compaction/reconciler.go

```diff
@@ -195,19 +198,27 @@ func (r *Reconciler) reconcileJob(ctx context.Context, logger logr.Logger, etcd
 
 	// Delete job and requeue if the job failed
 	if job.Status.Failed > 0 {
+		if job.Status.StartTime != nil {
+			CompactionJobDurationSeconds.With(prometheus.Labels{druidmetrics.LabelSucceeded: druidmetrics.ValueSucceededFalse}).Set(float64(time.Since(job.Status.StartTime.Time).Seconds()))
```

Comment:
```suggestion
			CompactionJobDurationSeconds.With(prometheus.Labels{druidmetrics.LabelSucceeded: druidmetrics.ValueSucceededFalse}).Set(time.Since(job.Status.StartTime.Time).Seconds())
```

Redundant `float64` type conversion.

Reply:
Done